### PR TITLE
FXIOS-1255 ⁃ Update issue templates

### PR DESCRIPTION
**Pull request by isabelrios**
┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1255)

### CODE REVIEW

**Comment by isabelrios** [comment]
@dnarcese not sure why by creating one template for intermittent issues the rest were modified... what would be the way to add that one?
I went to Settings -> Features -> Issues -> Set up templates and added one there... but there may be an easier way to do that... 
Thanks!

**Comment by dnarcese** [comment]
usually I go here https://github.com/mozilla-mobile/firefox-ios/tree/main/.github/ISSUE_TEMPLATE and add a file

File: .github/ISSUE_TEMPLATE/---feature.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
-name: "🎁 Feature request"
+name: "\U0001F381 Feature request"
 about: Suggest an idea for this project
-labels: "Feature 🎁"
```
Comment (dnarcese) [inline .github/ISSUE_TEMPLATE/---feature.md:4]: this should go back

File: .github/ISSUE_TEMPLATE/-broken_heart--intermittent-xcui-test-issue.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name: ":broken_heart: Intermittent XCUI Test Issue"
```
Comment (dnarcese) [inline .github/ISSUE_TEMPLATE/-broken_heart--intermittent-xcui-test-issue.md:2]: can this file be renamed without the `-broken_heart`

**Review by dnarcese — COMMENTED** [review]
Did you mean to add 3 new templates? They all look similar to me

File: .github/ISSUE_TEMPLATE/---feature.md (hunk 1 above)
Comment (isabelrios) [reply .github/ISSUE_TEMPLATE/---feature.md:4]: > Did you mean to add 3 new templates? They all look similar to me

No, I did not :( I think I may have messed up things here... I will close this PR and will try again...I just wanted to create one for Intermittent issues, don't know how the others were added/modified :(

**Pull request closed without merging**